src/Pact/Types/Crypto.hs

```diff
@@ -487,3 +406,30 @@ instance Serialize Ed25519.Signature where
   put (Ed25519.Sig s) = S.put s
   get = Ed25519.Sig <$> (S.get >>= S.getByteString)
 #endif
+
+
+-- | This type specifies the format of a WebAuthn signature.
```

Comment:
Do we want this buried here? I appreciate that it's not used outside of the module, but the _format_ will be widely used.

Reply:
I was thinking about making sure this format works for us internally, before moving it to a more prominent place and calling it out as something stable.